### PR TITLE
Was int counter

### PR DESCRIPTION
**Pull request by bourneash**
I added a node to count the places of an int. 

This combined with WAS Image Save can allow for a workflow that loads an image (ex: 9.png), img2img the loaded image, then save a 10.png (with WAS Image Save). The key feature of this is it allows for no padding of the input number (see a previous PR from me) when connected to the filename_number_padding input on WAS Image Save and the source int connected to a load file. Think of crude animation framing.

I can provide a more detailed use case for this if necessary, I'm sure others will be able to use it for something.

TLDR: It allows it to count past 9 without manually changing the filename_number_padding value manually.

### CODE REVIEW

**Comment by WASasquatch** [comment]
I'm not sure I *totally* follow, but it's not really intrusive on anything so, I'll allow it. Lol
